plugins/operators/model_evaluation/__init__.py

```diff
@@ -142,21 +140,20 @@ def get_subset_def_data_for_eval_key(
                 subset_def.get("view", ""),
             ]
         else:
+            if isinstance(subset_def, list):
+                subset_def = subset_def[0]
```

Comment:
what's the case when subset_def is a list?

Reply:
good question lanny - I should technically use the entire subset_def for the key. I will followup with next PR